src/solver/modeler/main.cpp

```diff
@@ -51,6 +54,27 @@ int main(int argc, const char** argv)
         logs.info() << "Libraries loaded";
         const auto system = LoadFiles::loadSystem(studyPath, libraries);
         logs.info() << "System loaded";
+
+        // Fillers, etc.
+        std::vector<Antares::Solver::Modeler::Api::LinearProblemFiller*> fillers;
+        // TODO memory
+        for (auto& [_, component]: system.Components())
+        {
+            fillers.push_back(new Antares::Optimization::ComponentFiller(component));
+        }
+
+        Antares::Solver::Modeler::Api::LinearProblemData LP_Data;
+        Antares::Solver::Modeler::Api::FillContext ctx = {0, 0};
+        Antares::Solver::Modeler::OrtoolsImpl::OrtoolsLinearProblem pb(false, parameters.solver);
```

Comment:
shouldn't it be MIP to handle booleans ?

Reply:
done